### PR TITLE
New plugin for NER (named entity recognition) using NTLK

### PR DESCRIPTION
**Pull request by JonanOribe**
Short script for NER searching over a user input

### CODE REVIEW

File: jarviscli/plugins/text_analysis.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import numpy
+import nltk
+from plugin import plugin
+from nltk import word_tokenize, sent_tokenize
+nltk.download('punkt')
```
Comment (pnhofmann) [inline jarviscli/plugins/text_analysis.py]: Should be downloaded in setup.sh. Compare line 74 setup.sh.

File: setup.sh [hunk 2]
```diff
@@ -72,6 +72,7 @@ pip install --upgrade -r requirements.txt
 
 # wordnet
 python -m nltk.downloader -d jarviscli/data/nltk wordnet
+pip install numpy
```
Comment (pnhofmann) [inline setup.sh]: Should be added to requirements.txt

=> Just add 'numpy' (alphabetically order) to requirements.txt file and and ``pip install --upgrade -r requirements.txt`` in setup.sh will install everything listed there.

File: jarviscli/plugins/text_analysis.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+nltk.download('averaged_perceptron_tagger')
+nltk.download('maxent_ne_chunker')
+nltk.download('words')
+
+
+@plugin("text analysis")
+def text_analysis(jarvis, s):
```
Comment (pnhofmann) [inline jarviscli/plugins/text_analysis.py]: Please add short description as docstring.

Like that =>
```
def text_analysis(jarvis, s):
    """
    HERE COMES YOUR DESCRIPTION
    """
```

File: jarviscli/plugins/text_analysis.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    sentence = jarvis.input("Write down the text: ")
+    tokens = nltk.word_tokenize(sentence)
+    tagged = nltk.pos_tag(tokens)
+    entities = nltk.chunk.ne_chunk(tagged)
+    print('Analysis result: ')
+    print(entities)
```
Comment (pnhofmann) [inline jarviscli/plugins/text_analysis.py]: Just as a proposal: Maybe better format output or add some kind of legend what e.g. 'WDT' means.

File: jarviscli/plugins/text_analysis.py (hunk 1 above)
Comment (JonanOribe) [reply jarviscli/plugins/text_analysis.py]: Could you, please, move the dependencies to the setup.sh for me?? i´m not sure how to manage this task

File: jarviscli/plugins/text_analysis.py (hunk 4 above)
Comment (JonanOribe) [reply jarviscli/plugins/text_analysis.py]: Ok, I will work around it

**Comment by appi147** [comment]
It seems that you have deleted your repo, please open a new PR.

**Pull request closed without merging**